src/ChainRulesCore.jl

```diff
@@ -22,6 +23,7 @@ include("differentials/abstract_differential.jl")
 include("differentials/abstract_zero.jl")
 include("differentials/thunks.jl")
 include("differentials/composite.jl")
+include("differentials/combinations.jl")
```

Comment:
I think this is the wrong location for this file.
It should be in the `src` directly, like `differential_arthmetic.jl`
Everything in `differentials/*.jl` focuses on one type and operations just on that type